### DISCUSSION TITLE
is it possible to have just the AR?

### DISCUSSION THREAD

**Discussion by felipefierro** [google/model-viewer#4315] [Q&A]
Hi! is it possible to have just the AR? like with a link or something, just having the button for the AR experience but not displaying the model before? thank you so much!

**Comment by yosun** [comment]
you can hijack the js and window.load() it?

**Comment by elalish** [comment]
See: https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-no3d - `reveal="manual"`.
